diff --git a/node_modules/instantsearch.js/es/lib/InstantSearch.js b/node_modules/instantsearch.js/es/lib/InstantSearch.js
index 142f37c..09e4f2a 100644
--- a/node_modules/instantsearch.js/es/lib/InstantSearch.js
+++ b/node_modules/instantsearch.js/es/lib/InstantSearch.js
@@ -90,7 +90,9 @@ function (_EventEmitter) {
     _defineProperty(_assertThisInitialized(_this), "middleware", []);
 
     _defineProperty(_assertThisInitialized(_this), "scheduleSearch", defer(function () {
-      _this.mainHelper.search();
+      if (_this.started) {
+        _this.mainHelper.search();
+      }
     }));
 
     _defineProperty(_assertThisInitialized(_this), "scheduleRender", defer(function () {
diff --git a/node_modules/instantsearch.js/es/widgets/index/index.js b/node_modules/instantsearch.js/es/widgets/index/index.js
index b6ce713..e587199 100644
--- a/node_modules/instantsearch.js/es/widgets/index/index.js
+++ b/node_modules/instantsearch.js/es/widgets/index/index.js
@@ -312,6 +312,11 @@ var index = function index(props) {
 
       helper.on('change', function (_ref4) {
         var state = _ref4.state;
+
+        if (!instantSearchInstance.started) {
+          return;
+        }
+
         localUiState = getLocalWidgetsState(localWidgets, {
           searchParameters: state,
           helper: helper
